src/Input/index.tsx

```diff
@@ -7,9 +7,9 @@ export const Input = ({
   autoFocus = false,
   label = '',
   style = {},
-  onChange,
-  onFocus,
-  onBlur,
+  onChange = () => {},
+  onFocus = () => {},
+  onBlur = () => {},
   defaultValue = undefined,
   value = undefined,
```

Comment:
I think without the ` = undefined` assignment its undefined too.

Haha thats on me originally 😊